src/js/services/pull-request-storage-accessor.ts

```diff
@@ -30,6 +31,36 @@ const PullRequestStorageAccessor = () => {
     return data[key] ? JSON.parse(data[key]) : [];
   };
 
+  const syncIgnoredPrs = async (record: PullRequestRecord): Promise<Partial<PullRequest>[]> => {
+    const ignoredPrsString = await SettingsStorageAccessor().loadIgnoredPrs();
+    if (!ignoredPrsString.length) {
+      return [];
+    }
+
+    const ignoredPrs = ignoredPrsString.split(',').map(parsePullRequest);
```

Comment:
I'd always strip the split results because people then to enter whitespace.

Reply:
right, good point!